### ISSUE TITLE
nq-distilbert-base-v1

### ISSUE THREAD

**Issue by marjanhs** [UKPLab/sentence-transformers#1122]
Hello, 
Thank you for the great resources ypu have provided. 
Could you elaborate on how the `nq-distilbert-base-v1` has been trained on NQ dataset ?
Is it trained with semantic similarity perspective, maximizing the similarity of questions and answer pairs,   or  with the retrieval perspective similar to the DPR model or ...? what was the objective function ?
Thanks

**Comment by nreimers** [comment]
The loss was MultipleNegativesRankingLoss, so similarity between question and answer was optimized. You can use it for retrieval, where it greatly outperforms DPR.

**Comment by marjanhs** [comment]
thanks for the quick answer. What about for `msmarco-distilbert-base-v3` ? the same loss ?

**Comment by nreimers** [comment]
Yes

**Comment by guillaume-be** [comment]
Thank you @nreimers ,

Could you provide some additional details on the training parameters for `nq-distilbert-base-v1`? I've been looking for details regarding the learning rate, number of epochs, weights decay but could not find any reference to the parameters used
